scss/_reboot.scss

```diff
@@ -350,21 +350,52 @@ sup { top: -.5em; }
 // Links
 
 a {
-  color: rgba(var(--#{$prefix}link-color-rgb), var(--#{$prefix}link-opacity, 1));
+  // OUDS mod
+  --#{$prefix}link-color: #{$ouds-link-color-content-enabled};
+  --#{$prefix}link-hover-color: #{$ouds-link-color-content-hover};
+  --#{$prefix}link-focus-color: #{$ouds-link-color-content-focus};
+  --#{$prefix}link-active-color: #{$ouds-link-color-content-pressed};
+  --#{$prefix}link-disabled-color: #{$ouds-color-action-disabled};
+  --#{$prefix}link-visited-color: #{$ouds-color-action-visited};
+  // End mod
+
+  font-weight: $link-font-weight; // OUDS mod
+  color: var(--#{$prefix}link-color);
   text-decoration: $link-decoration;
 
+  .visited-links &,
+  &.visited-links {
+    color: var(--#{$prefix}link-visited-color);
+  }
+
```

Comment:
This part should be applied only on the links with a`:visited` state, I don't think it's the case